src/modules/poweraccent/PowerAccent.Core/PowerAccent.cs

```diff
@@ -13,15 +17,26 @@ public class PowerAccent : IDisposable
 {
     private readonly SettingsService _settingService;
 
+    // Keys that show a description (like dashes) when ShowCharacterInfoSetting is 1
+    private readonly LetterKey[] _specialLetterKeys = new LetterKey[] { LetterKey.VK_O };
```

Comment:
Let's have a better name that contains the word description to make the name more clear.

Something like `_letterKeysShowingDescrition`